samples/bluetooth/mesh/ble_peripheral_lbs_coex/src/lb_service_handler.c

```diff
@@ -119,7 +119,7 @@ static void lbs_adv_start(void)
 {
 	int err;
 	size_t id_count = 0xFF;
-
+	struct bt_le_adv_param adv_params = *BT_LE_ADV_CONN;
```

Comment:
```suggestion
	struct bt_le_adv_param *adv_params = *BT_LE_ADV_CONN;
```

BT_LE_ADV_CONN creates a 'struct', so its reference needs to be held into corresponding pointer.

Reply:
https://github.com/nrfconnect/sdk-zephyr/blob/main/doc/releases/migration-guide-3.7.rst?plain=1#L804-L810
we need to update id so our case is the second.

Reply:
Right.